src/jit/instr.h

```diff
@@ -284,11 +284,14 @@ END_DECLARE_TYPED_ENUM(emitAttr,unsigned)
 #define EmitSize(x)                 (EA_ATTR(genTypeSize(TypeGet(x))))
 
 // Enum specifying the instruction set for generating floating point or SIMD code.
+// These enums are ordered such that each one is inclusive of previous instruction sets.
 enum InstructionSet
```

Comment:
how does "InstructionSet_NONE" fit in, then? Maybe you should comment that, too.